build.txt

```diff
@@ -1,3 +1,4 @@
 displayName = The Bereft Souls
 author = Brome
-version = 0.1
\ No newline at end of file
+version = 0.1
+modReferences = CalamityMod, ThoriumMod, SOTS
```

Comment:
We prefer having WeakReferences wherever possible, this would only be acceptable if there's no way around it. Additionally make sure to tag every mod with a minimum version like so: `ModName@1.0`/`ModName@1.x`/ModName@2.0`/ModName@2.x`/ModName@x.x` (x being a digit and the version digit being valid).

For more information: [tModLoader/wiki/Expert-Cross-Mod-Content#strong-and-weak-references](https://github.com/tModLoader/tModLoader/wiki/Expert-Cross-Mod-Content#strong-and-weak-references).